docs/guides/node-operator-manual.md

```diff
@@ -105,23 +105,43 @@ a Node Operator submits a set of the corresponding signatures [as defined in the
 The fork version used for generating the signature must correspond to the fork version of the Beacon
 chain the instance of Lido protocol is targeted to.
 
+#### Mainnet
+
+Make sure to obtain a correct withdrawal address by finding it inside active withdrawal credentials either on Aragon UI or calling the contract via [`Lido.getWithdrawalCredentials()`]. You can quickly find on the [Etherscan page for the Mainnet-deployed Lido].
+
+For example withdrawal credentials `0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f` mean that the withdrawal address is `0xb9d7934878b5fb9610b3fe8a5e441e8fad7e293f`. Always verify the address is correct using an [explorer] - you will see it's deployed from the Lido deployer.
+
+#### Testnet
+
 You can obtain the protocol withdrawal credentials by calling [`Lido.getWithdrawalCredentials()`].
 On the [Etherscan page for the Prater-deployed Lido], it’s the field number 19. The ABI of the
 `Lido` contract can be found in [`lib/abi/Lido.json`].
 
 [bls12-381]: https://ethresear.ch/t/pragmatic-signature-aggregation-with-bls/2105
 [as defined in the spec]: https://github.com/ethereum/annotated-spec/blob/master/phase0/beacon-chain.md#depositmessage
+[explorer]: https://etherscan.io/address/0xb9d7934878b5fb9610b3fe8a5e441e8fad7e293f
 [`lido.getwithdrawalcredentials()`]: https://github.com/lidofinance/lido-dao/blob/971ac8f/contracts/0.4.24/Lido.sol#L312
+[etherscan page for the mainnet-deployed lido]: https://etherscan.io/address/0xae7ab96520de3a18e5e111b5eaab095312d7fe84#readProxyContract
 [etherscan page for the prater-deployed lido]: https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F#readProxyContract
 [`lib/abi/lido.json`]: https://github.com/lidofinance/lido-dao/blob/971ac8f/lib/abi/Lido.json
 
-#### Using the forked eth2.0-deposit-cli
+#### Using eth2.0-deposit-cli
+
+##### Mainnet
+
+For a mainnet deployment, use the latest release of [`eth2.0-deposit-cli`].
```

Comment:
Is there a way to used pre-defined withdrawal pubkey with the non-forked cli app? The difference between the mainnet and the testnet flow seems a bit off

Reply:
I don't think there is - we actually forked it for this exact reason :)

Non-forked eth2-deposit-cli can take only `eth1_withdrawal_address` as an option, otherwise it will use a mnemonic-derived withdrawal pubkey.

Reply:
And we don't really have a clean way to get the withdrawal address for node ops. Our contract has a view function to get the whole withdrawal_credentials with withdrawal address embedded, thus need easy but manual extraction. This is far from ideal, I completely agree, but can't think of any way to improve this.